static/js/popovers.js

```diff
@@ -819,6 +819,9 @@ export function register_click_handlers() {
         if (overlays.settings_open()) {
             overlays.close_overlay("settings");
         }
+        if (overlays.streams_open()) {
+            overlays.close_overlay("subscriptions");
```

Comment:
Can we instead call `close_active`? We would want to close any opened overlay here when going to message view.

Reply:
> Can we instead call `close_active`? We would want to close any opened overlay here when going to message view.

Updated the PR according to this.